Update puppet scripts for OB200

DoryFazna opened the pull request:
## Purpose
To Run 200 obiam, obam , obbi for UK,Berlin, AU.

## Goals
To run and test OB-2.0.0 products.

## Approach
Compared OB200 scripts and configs and updated the changes.
Tested with all specs.

## User stories
> Summary of user stories addressed by this change>

## Release note
> Brief description of the new feature or bug fix as it will appear in the release notes

## Documentation
> Link(s) to product documentation that addresses the changes of this PR. If no doc impact, enter “N/A” plus brief explanation of why there’s no doc impact

## Training
> Link to the PR for changes to the training content in https://github.com/wso2/WSO2-Training, if applicable

## Certification
> Type “Sent” when you have provided new/updated certification questions, plus four answers for each question (correct answer highlighted in bold), based on this change. Certification questions/answers should be sent to certification@wso2.com and NOT pasted in this PR. If there is no impact on certification exams, type “N/A” and explain why.

## Marketing
> Link to drafts of marketing content that will describe and promote this feature, including product page changes, technical articles, blog posts, videos, etc., if applicable

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

## Samples
> Provide high-level details about the samples related to this feature

## Related PRs
> List any other related PRs

## Migrations (if applicable)
> Describe migration steps and platforms on which migration has been tested

## Test environment
> List all JDK versions, operating systems, databases, and browser/versions on which this feature/fix was tested
 
## Learning
> Describe the research phase and any blog posts, patterns, libraries, or add-ons you used to solve the problem.

DivyaPremanantha reviewed (changes requested):
Please do the changes required for the readme file and @isharailanga please review that

Dinushi commented:
WUM updates for UK MI Reporting siddhi files for UK 3.1.5 were also released. Shall we create a issue to track this requirement to add UK 3.1.5 siddhi-files also to puppet.

isharailanga commented:
> Please do the changes required for the readme file and @isharailanga please review that

Hi @DivyaPremanantha @DoryFazna,

Ack. Please keep me posted.

DoryFazna commented:
> WUM updates for UK MI Reporting siddhi files for UK 3.1.5 were also released

Created the issue. Thanks

DoryFazna commented:
> Please do the changes required for the readme file and @isharailanga please review that

Updated. Thanks.

> > Please do the changes required for the readme file and @isharailanga please review that
> 
> Hi @DivyaPremanantha @DoryFazna,
> 
> Ack. Please keep me posted.

Updated the README file. Please check.

isharailanga commented:
> > > Please do the changes required for the readme file and @isharailanga please review that
> > 
> > 
> > Hi @DivyaPremanantha @DoryFazna,
> > Ack. Please keep me posted.
> 
> Updated the README file. Please review.

Line 26 in Readme:
Add file and use specification instead of Spec as follows:
Update the `<puppet_environment>/modules/ob_common/manifests/params.pp` **file** of **Puppetmaster** according to the Open Banking **specification** (UK, AU, Berlin). The default value is set as UK.

isharailanga commented on README.md:
````diff
@@ -97,7 +97,7 @@ Each Puppet module contains the following ```.pp``` files.
 ## Customizations
 
-1.  When updating the `<puppet_environment>/modules/ob_common/manifests/params.pp` of **Puppetmaster** to point the created databases, 
+1.  When updating the `<puppet_environment>/modules/ob_common/manifests/params.pp` file of **Puppetmaster** to point the created databases, 
````
We can get rid of the numbering for this section

DoryFazna replied:
Done

isharailanga reviewed (approved):
Approving the README.md file.

Dinushi commented:
There is few more updates needs to be done to the following template file.

https://raw.githubusercontent.com/wso2/puppet-open-banking/master/modules/obam/templates/carbon-home/repository/resources/api_templates/velocity_template.xml.erb 

There are few new changes in WUM related to "cds-admin" in the velocity_template.xml. Please add those changes as well.

Dinushi reviewed (changes requested):
There is few more updates needs to be done to the following template file.

https://raw.githubusercontent.com/wso2/puppet-open-banking/master/modules/obam/templates/carbon-home/repository/resources/api_templates/velocity_template.xml.erb

There are few new changes in WUM related to "cds-admin" in the velocity_template.xml. Please add those changes as well.

DoryFazna commented:
> There is few more updates needs to be done to the following template file.
> 
> https://raw.githubusercontent.com/wso2/puppet-open-banking/master/modules/obam/templates/carbon-home/repository/resources/api_templates/velocity_template.xml.erb
> 
> There are few new changes in WUM related to "cds-admin" in the velocity_template.xml. Please add those changes as well.

Updated. Thanks.
